Example/src/screens/Menus.tsx

```diff
@@ -13,18 +13,119 @@ import {
     UIBottomSheet,
     UIFullscreenSheet,
     UIPopup,
+    UIActionSheetContainerChildType,
     useIntrinsicSizeScrollView,
 } from '@tonlabs/uikit.popups';
 import { ScrollView } from '@tonlabs/uikit.scrolls';
 import { UIMaterialTextView } from '@tonlabs/uikit.inputs';
 import { UIBoxButton, UILinkButton } from '@tonlabs/uikit.controls';
 import { UILabel, ColorVariants, useTheme } from '@tonlabs/uikit.themes';
+import { UIAssets } from '@tonlabs/uikit.assets';
 import { UIPinCode, UIPinCodeBiometryType } from '@tonlabs/uicast.pin-code';
 import { UICountryPicker } from '@tonlabs/uicast.country-picker';
 
 import { ExampleSection } from '../components/ExampleSection';
 import { ExampleScreen } from '../components/ExampleScreen';
 
+const onPress = () => {
+    console.log('Action');
+};
+
+const foregroundList: UIActionSheetContainerChildType[] = [
+    <UIPopup.ActionSheet.CustomAction key="1">
+        <UIPopup.ActionSheet.PrimaryColumn>
```

Comment:
Can we somehow accept single action and wrap it in PrimaryColumn if not other options aren't provided? 🤔 Looks too verbose for now

Reply:
Yes, we can. We don't need PrimaryColumn for this purpose. https://github.com/tonlabs/UIKit/pull/1461/files#diff-cfed5573f8056947bb7e2760d99ca09c630ecea6b3df46283987bb8711305632R369-R373

Reply:
Either link is broken or I just doesn't understand explanation :)

Reply:
The link was broken :) You can see example of using simple action in bottom of this file (`UIPopup.ActionSheet.Action`)

Reply:
Got it :) But actually I meant the Foreground api. Like that next one would be valid:
```ts
<UIPopup.ActionSheet.CustomAction>
  <UIPopup.ActionSheet.Action title="Press me" />
</UIPopup.ActionSheet.CustomAction>
```